doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+-->
+
+FIWARE-NGSI Simple API (v2) Cookbook.
+
+This is a cookbook with recipes that show how to use FIWARE-NGSI v2 aka FIWARE-NGSI Simple API.
+Use this examples for learning and enjoy!
+
+This is a work in progress and is changing on a daily basis.
+Please send your comments to fiware-ngsi@lists.fiware.org
+
+# FIWARE-NGSI-Simple API Root [/v2]
+
+This resource allows you to obtain a list of subresources published by the API:
+
+* entities to deal with your NGSI entities
+* types to deal with your entity types
+* subscriptions to deal with subscriptions to context changes
+* registrations to deal with context data by registering external providers
+
+## Retrieve the Entry Point [GET]
+
++ Response 200 (application/json)
+
+        {
+            "entities_url":      "/v2/entities",
+            "types_url":         "/v2/types",
+            "subscriptions_url": "/v2/subscriptions",
+            "registrations_url": "/v2/registrations"
+        }
+
+## Listing context entities [/v2/entities{?type,id,idPattern,limit,offset}]
+
+### List all entities [GET /v2/entities{?limit,offset}]
+
++ Parameters
+    + limit: 20 (optional, number) - Limit the number of entities to be retrieved
+    + offset: 20 (optional, number) - Skip a number of records. Pagination
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         },
+         {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": 22.5
+         },
+         {
+            "type": "Car",
+            "id": "P-9873-K",
+            "speed": {
+                "value": 100,
+                "type": "number",
+                "accuracy": 2,
+                "timestamp": {
+                    "value": "2015-06-04T07:20:27.378Z",
+                    "type": "date"
+                }
+            }
+         }
+        ]
+
+### List entities by id [GET /v2/entities{?id}]
+
++ Parameters
+    + id: DC_S1_D41 (optional, string) - Only retrieve entities with this id
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         }
+        ]
+
+### List entities by list of ids [GET /v2/entities{?id}]
+
++ Parameters
+    + id: DC_S1_D41, Boe_Idearium (optional, string) - Id list with the entities to be retrieved
+
++ Response 200 (application/json)
+
+        [
+         {
+            "type": "Room",
+            "id": "DC_S1_D41",
+            "temperature": 35.6
+         },
+         {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": 22.5
+         }
+        ]
+
+### List all entities of an specific type [GET /v2/entities{?type}]
```

Comment:
What about an operation for "List entities by list of types"? Taking into account that "List entities by list of ids" has been included, I thinks that "List entities by list of types" would make sense.